src/System.Memory/src/System.Memory.csproj

```diff
@@ -16,9 +16,6 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netcoreapp-Windows_NT-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="System\Pinnable.cs" />
```

Comment:
Pinnable is only used by the slow span, so I think it needs to be factored out such that it's only included in the slow span assembly.